### PR TITLE
Add script for syncing data to the docs repository

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This new script copies the shiny new Seed CRD example to the docs repo, as well as updating the Prometheus runbook based on our current ruleset.

The Seed CRD YAML file can then be imported by whatever content page wants to. This way we don't hardcode any document structure into this script.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: api/hack/ci/ci-update-docs.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# copy interesting files over
+cp ../docs/zz_generated.seed.yaml data/seed.yaml
+
+# re-create Prometheus runbook
+make runbook
+
+# update repo
+git add .
+git commit -m "update data files to kubermatic@$REVISION"
+git push
```
Comment (alvaroaleman) [inline api/hack/ci/ci-update-docs.sh:29]: Can you comment the `git push` so we can check if this actually works?

Comment (xrstf) [reply api/hack/ci/ci-update-docs.sh:29]: Can do.

**Comment by alvaroaleman** [comment]
Lookgs good! Re-revert the change to not push and merge?

**Comment by xrstf** [comment]
Let me run the new job once again to see the diff.

**Comment by xrstf** [comment]
I'm now satisfied, this is ready for review/approval.

**Pull request merged**